src/_utils/useFocusTrap/index.unit.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { useRef } from 'react'
+import createFocusTrap from 'focus-trap'
+import { mount } from 'enzyme'
+import { useFocusTrap } from './index'
+
+let TestComponent
+const activate = jest.fn()
+const deactivate = jest.fn()
+
+describe('useFocusTrap', () => {
+  const onClose = jest.fn()
+
+  const mockFocusTrap = createFocusTrap as jest.Mock
+  mockFocusTrap.mockReturnValue({
+    activate,
+    deactivate,
+  })
+  beforeEach(() => {
+    TestComponent = () => {
+      const ref = useRef<HTMLDivElement>(null)
+      useFocusTrap(ref, onClose)
+      return (
+        <div ref={ref}>
+          <button type="button">1</button>
+          <button type="button">2</button>
+          <button type="button">3</button>
+        </div>
+      )
+    }
+  })
+
+  it('Should activate focus trap', () => {
+    mount(<TestComponent />)
+    expect(createFocusTrap).toHaveBeenCalled()
+    expect(activate).toHaveBeenCalled()
```

Comment:
Can we also check that the first button has the focus?

Reply:
I don't want to test focus-trap lib, I just want to check that the correct methods are called.

Reply:
I don't see it as testing the focus-trap lib but as testing the functionality of our component. The fact that we use a lib is an implementation detail.

Reply:
I'll try, I'll remove the mock and check if it's possible

Reply:
As seen together we can't test this due to the test environment and a condition in focus-trap lib on the .offsetParent (which is null in our test)